src/Callback.php

```diff
@@ -105,43 +88,26 @@ public function init(): void
      */
     public function set($callback, $args = [])
     {
-        if ($this->postTrigger) {
-            if (isset($_POST[$this->postTrigger])) {
-                $this->app->catch_runaway_callbacks = false;
-                $this->triggered = $_POST[$this->postTrigger];
-
-                $t = $this->app->run_called;
-                $this->app->run_called = true;
-                $ret = call_user_func_array($callback, $args);
-                $this->app->run_called = $t;
-
-                return $ret;
-            }
-        } else {
-            if (isset($_GET[$this->urlTrigger])) {
-                $this->app->catch_runaway_callbacks = false;
-                $this->triggered = $_GET[$this->urlTrigger];
-
-                $t = $this->app->run_called;
-                $this->app->run_called = true;
-                $this->owner->stickyGet($this->urlTrigger);
-                $ret = call_user_func_array($callback, $args);
-                //$this->app->stickyForget($this->name);
-                $this->app->run_called = $t;
-
-                return $ret;
-            }
+        if ($this->isTriggered()) {
+            $this->owner->stickyGet($this->urlTrigger);
```

Comment:
I think we should not registed sticky GET even inside owner, this can/will be issue if the owner contains two Callbacks and at least one directly

I think we do not need this at all - sticky GET set in self::getUrlArguments()

Reply:
Yes this is need, otherwise triggering element inside a callback will not run, i.e. callback in callback will not work without it.

Reply:
then we need to move everything related with Url inside View inside a trait or make Callback finally an instance of View :)